internal/dwarf/frame/table.go

```diff
@@ -219,7 +219,10 @@ func executeCIEInstructions(cie *CommonInformationEntry) *FrameContext {
 		instructions: frames,
 		buf:          bytes.NewBuffer(initialInstructions),
 	}
-	//frame.executeDwarfProgram()
+	//TODO: Uncommenting this as this gets us the correct first row
```

Comment:
@v-thakkar I think this should be there regardless of changes in `executeDWARFProgram` since this is to parse the CIE.